### PR TITLE
add button to redirect to results page

### PR DESCRIPTION
**Pull request by eng-esther**
remove absolute
Need to discuss with team first before moving the button above the search input field

### CODE REVIEW

**Review by kimberlythegeek — APPROVED** [review]
looks good--thanks!

**Comment by julienw** [comment]
Hey @eng-esther ! Can you please clean up the commit log? currently it contains the message in your former latest commit, so it doesn't reflect the content of this patch.

Also I see there are a lot of conflicting files, it would be good to rebase the patch before we can review. Most of them are snaphot files so these ones should be as easy as running the test runner with `-u`.

Thanks!

**Comment by julienw** [comment]
I also see lint errors in this patch when I try to run it locally, can you please take care to fix them? Thanks so much :-)

File: src/components/Search/SearchView.tsx [hunk 1]
```diff
@@ -40,6 +49,19 @@ function SearchView(props: SearchViewProps) {
...
       </Grid>
+
+      <Grid item className="compare-button-section">
+          {selectedRevisions.length > 0 && (
+            <Button
+              className="compare-button"
+              variant="contained"
+              onClick={goToResultsPage}
+            >
+              COMPARE
```
[the remark this answers is not in the document]
Comment (eng-esther) [reply src/components/Search/SearchView.tsx:60]: > I also see lint errors in this patch when I try to run it locally, can you please take care to fix them? Thanks so much :-)
> 
> **update:** I don't see them when running the linter, only when running the dev enviromnent: image
> 
> So not sure what's happening here... Maybe you can ignore my comment for now.

Hi Julien, can't reproduce on my local, will rebase tho.

**Review by beatrice-acasandrei — APPROVED** [review]

File: src/components/Search/SearchView.tsx [hunk 2]
```diff
@@ -15,8 +18,16 @@ import SearchViewInit from './SearchViewInit';
...
 
+  const navigate = useNavigate();
+
+  const goToResultsPage = () => {
+    navigate('/results', { replace: false });
+  };
+
   const { searchResults, selectedRevisions } = props;
 
+  const compare = 'compare';
```
Comment (julienw) [inline src/components/Search/SearchView.tsx:29]: I'm just wondering why you added this variable?

Comment (eng-esther) [reply src/components/Search/SearchView.tsx:29]: messaged you.

File: src/theme/components.js [hunk 3]
```diff
@@ -11,6 +11,12 @@ const components = {
           lineHeight: '1.4375em',
           padding: '16.5px 14px',
         },
+        '&.compare-button': {
+          lineHeight: '1.4375em',
+          padding: '16.5px 14px',
+          textTransform: 'uppercase',
```
Comment (eng-esther) [inline src/theme/components.js:17]: @julienw added this as adviced, (similar question to what I direct messaged you).
Is this the way you were suggestng? If not did you have a better way in mind ?

**Review by julienw — APPROVED** [review]
lgtm!

**Pull request merged**